src/openapi_python_generator/language_converters/python/templates/aiohttp.jinja2

```diff
@@ -24,6 +24,7 @@ async def {{ operation_id }}({{ params }}) -> {% if return_type.type is none or
                 raise HTTPException(inital_response.status, f'{{ operationId }} failed with status code: {inital_response.status}')
             response = await inital_response.json()
 
+{% if return_type.type is not none and return_type.type.converted_type is not none %}
```

Comment:
I don't like the nesting here, it makes things a bit confusing, and the template already is. I suggest to do the following:

```
{% if return_type.type is  none or return_type.type.converted_type is  none %}
    return None
{%elif return_type.complex_type %}
.
.
.
```


Same for all other Templates

Reply:
Good point. I pushed some changes to improve the readability.